kornia/augmentation/container/augment.py

```diff
@@ -242,13 +237,18 @@ def inverse(  # type: ignore
                 input = arg.data  # all boxes are in (B, N, 4, 2) format now.
             else:
                 input = arg
-            for (name, module), param in zip_longest(list(self.get_forward_sequence(params))[::-1], params[::-1]):
+            for (_, module), param in zip_longest(list(self.get_forward_sequence(params))[::-1], params[::-1]):
                 if isinstance(module, (_AugmentationBase, ImageSequential)):
-                    param = params[name] if name in params else param
+                    # TODO(jian): verify what's happening here. Mypy was complaining,
+                    # param = params[name] if name in params else param
```

Comment:
@shijianjian please check this — seems to pass test but not confident what’s happening

Reply:
This shouldn't pass. It consumes wrong param but still reproducible I guess.

Reply:
Params is typed as list but in practice seems to be a dict